pkgs/applications/networking/browsers/ladybird/default.nix

```diff
@@ -139,6 +135,8 @@ stdenv.mkDerivation (finalAttrs: {
   postInstall = lib.optionalString stdenv.hostPlatform.isDarwin ''
     mkdir -p $out/Applications $out/bin
     mv $out/bundle/Ladybird.app $out/Applications
+    # copy skia to the bundle - there's gotta be a better way to do this...
```

Comment:
I tried using fixDarwinDylibNames to get the rpath for libskia.dylib working but couldn't figure it out.

Reply:
I don't know darwin so I can't help much. If it works that's enough for me.

How did you try to use fixDarwinDylibNames?

Reply:
I tried adding it to the nativeBuildInputs hook like I saw in a lot of other packages but it didnt seem to work. Looking more at the docs and how the hook is being used, I added to hook to skia instead of ladybird and now the application starts up properly without copying the skia binary to the app bundle.

<img width="1094" alt="Screenshot 2024-12-23 at 12 13 38 PM" src="https://github.com/user-attachments/assets/8dc4c78a-a20d-4503-867a-bae68e0cddb4" />